src/battlearena/SearchPlayers.js

```diff
@@ -30,11 +31,39 @@ class SearchPlayers extends React.Component<Props, State> {
 		this.handleKeyPress.bind(this);
 
 		this.state = {
-			playerList: ["Jim", "John", "Jimfrey", "Eich", "Eichers", "Baylor", "Jorge", "David", "Emil", "Adam"], // NEEDS API CALL HERE
+			playerList: [], // NEEDS API CALL HERE

```

Comment:
We need to store more than just the user here. Can you make a user class and make this an array of users?

Reply:
Ok, I'll get that done now.